setup.py

```diff
@@ -16,6 +23,8 @@ def get_version():
     name='extruct',
     version=get_version(),
     description='Extract embedded metadata from HTML markup',
+    long_description=get_readme(),
+    long_description_content_type='text/markdown',
```

Comment:
Since README is in RST, I think the type should be different, according to https://packaging.python.org/guides/making-a-pypi-friendly-readme/#including-your-readme-in-your-package-s-metadata
```suggestion
    long_description_content_type='text/x-rst',
```

Reply:
Good catch, thanks!